test/selenium/src/lib/entities/entities_factory.py

```diff
@@ -222,12 +211,7 @@ def create(cls, title=None, id=None, href=None, type=None,
     """
     ca_entity = cls._create_random_ca()
     ca_entity = cls._update_ca_attrs_values(
-        obj=ca_entity, is_allow_none_values=False, title=title, id=id,
-        href=href, type=type, definition_type=definition_type,
-        attribute_type=attribute_type, helptext=helptext,
-        placeholder=placeholder, mandatory=mandatory,
-        multi_choice_options=multi_choice_options, updated_at=updated_at,
-        modified_by=modified_by, created_at=created_at)
+        obj=ca_entity, is_allow_none_values=False, **attrs)
```

Comment:
as previous, good!

Reply:
I have a draft with a new model ready to ACL and RBAC and with different types of improvements! Changes coming soon:)